tritonserver/samples/pom.xml

```diff
@@ -21,6 +21,28 @@
         </dependency>
     </dependencies>
     <build>
-        <sourceDirectory>.</sourceDirectory>
```

Comment:
We need to leave that line because the source file isn't in the default directory.

Reply:
Yes, added back this line.